src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas1/Cas1SpaceBookingTest.kt

```diff
@@ -657,6 +681,51 @@ class Cas1SpaceBookingTest {
       assertThat(response[4].tier).isEqualTo("A")
       assertThat(response[4].person.crn).isEqualTo("CRN_CURRENT1")
     }
+
+    @Order(9999)
```

Comment:
need to ensure that this test runs last due to data setup breaking other tests otherwise. Not sure if this is ideal and can investigate tearing down data at end of each test also. Wanted to avoid increasing pipeline run time further with teardown.
Not sure if this is the right solution though so please feed in!

Reply:
this sounds good! I'll have a crack at it, thanks